django_mongodb/expressions.py

```diff
@@ -70,6 +70,9 @@ def value(self, compiler, connection):  # noqa: ARG001
     elif isinstance(value, datetime.date):
         # Turn dates into datetimes since BSON doesn't support dates.
         value = datetime.datetime.combine(value, datetime.datetime.min.time())
+    elif isinstance(value, datetime.timedelta):
+        # DurationField store milliseconds rather than microseconds.
+        value = value.total_seconds() * 1000
     return {"$literal": value}
```

Comment:
🤔 I am wondering if `(valie / timedelta(millisecond=1)).total_seconds()` works?

Reply:
Yes, except that `timedelta / timedelta` gives a float, so no need for `total_seconds()`. Do you see an advantage to that version?

Reply:
Agree on no need of total_second I misunderstood the docs. https://docs.python.org/3/library/datetime.html#datetime.timedelta.total_seconds
But it recommended, by the docs, to use the division by time delta. I see no advantages, only the docs